build/nogo_config.json

```diff
@@ -295,6 +295,93 @@
+      "br/pkg/lightning/errormanager/errormanager_test.go": "ignore errormanager code",
+      "br/pkg/lightning/backend/kv/sql2kv_test.go": "ignore sql2kv code",
+      "br/pkg/stream/rewrite_meta_rawkv_test.go": "ignore rewrite_meta_rawkv code",
+      "br/pkg/lightning/lightning.go": "ignore lightning code",
+      "br/pkg/lightning/importer": "ignore importer code",
+      "br/cmd/tidb-lightning-ctl/main.go": "ignore main code",
+      "br/pkg/stream/rewrite_meta_rawkv.go": "ignore rewrite_meta_rawkv code",
+      "br/pkg/lightning/web/": "ignore web code",
+      "br/pkg/mock/": "ignore mock code",
+      "br/pkg/aws/ebs.go": "ignore ebs code",
+      "br/pkg/backup/": "ignore backup code",
+      "br/pkg/restore/": "ignore restore code",
+      "br/pkg/task/": "ignore task code",
+      "br/pkg/version/": "ignore version code",
+      "ddl/util/util.go": "ignore util code",
+      "ddl/syncer/syncer.go": "ignore syncer code",
+      "ddl/ingest/backend.go": "ignore backend code",
+      "ddl/ingest/backend_mgr.go": "ignore backend_mgr code",
+      "ddl/ingest/engine_mgr.go": "ignore engine_mgr code",
+      "util/column-mapping/column.go": "ignore error",
+      "parser/mysql/": "ignore mysql code",
+      "parser/auth": "ignore auth code",
+      "parser/charset/charset.go": "ignore charset code",
+      "parser/types/field_type.go": "ignore field_type code",
+      "parser/model/ddl.go": "ignore ddl code",
+      "parser/model/model.go": "ignore model code",
+      "parser/model/model_test.go": "ignore tables code",
+      "parser/ast": "ignore ast code",
+      "parser/parser.go": "ignore parser code",
+      "parser/hintparser.go": "ignore hintparser code",
+      "parser/yy_parser.go": "ignore yy_parser code",
+      "parser/digester.go": "ignore digester code",
+      "parser/consistent_test.go": "ignore consistent code",
+      "parser/bench_test.go": "ignore bench code",
+      "parser/parser_test.go": "ignore parser_test code",
+      "parser/digester_test.go": "ignore digester_test code",
+      "parser/hintparser_test.go": "ignore hintparser_test code",
+      "errno/errname.go": "ignore errname code",
+      "planner/funcdep/fd_graph.go": "ignore fd_graph code",
+      "planner/funcdep/doc.go": "ignore funcdep code",
+      "planner/util/path.go": "ignore path code",
+      "planner/memo/": "ignore memo code",
+      "ttl/cache/ttlstatus.go": "ignore ttlstatus code",
+      "ttl/ttlworker/": "ignore ttlworker code",
+      "ttl/sqlbuilder/sql_test.go": "ignore sql_test code",
+      "planner/core/": "ignore core code",
+      "planner/optimize.go": "ignore optimize code",
+      "planner/cascades/": "ignore cascades code",
+      "planner/funcdep/extract_fd_test.go": "ignore extract_fd code",
+      "planner/funcdep/only_full_group_by_test.go": "ignore only_full_group_by code",
+      "dumpling/export": "ignore export code",
+      "ddl/placement/": "ignore placement code"
+    },
+    "only_files": {
```

Comment:
Why is it only for these directories? If the left work is for following PRs, this one should not close the issue.

Reply:
Okay. Then it should ref #42570 instead of closing it.